feeder/api/models/kronos.py

```diff
@@ -19,18 +19,18 @@ class NewGateway(BaseModel):
     }
     """
 
-    name: str
-    uid: str
-    osName: str
-    type: str
-    softwareName: str
-    softwareVersion: str
-    sdkVersion: str
+    name: Optional[str]
+    uid: Optional[str]
+    osName: Optional[str]
+    type: Optional[str]
+    softwareName: Optional[str]
+    softwareVersion: Optional[str]
+    sdkVersion: Optional[str]
 
 
 class Gateway(NewGateway):
     hid: str
-    pri: str
+    pri: Optional[str]
```

Comment:
PRI is just a formatted string of the HID, what is the goal in making this optional?

Reply:
Sure, this could be stored. Although that begs the question why store it if it's able to be reconstructed from the `hid`?

Reply:
We aren't storing it. It is [added to the model by the API route](https://github.com/tedder/petnet-feeder-service/blob/master/feeder/api/routers/kronos.py#L31) before it is serialized and returned to the user.

Reply:
Oh, I'll add that back, then.